src/metaAnalysis.ts

```diff
@@ -2,28 +2,31 @@ import { AirtableRecord } from "./types"
 
 const Z_SCORE: number = 1.96;
 
-function calcPooledPrevalence(records: AirtableRecord[]) {
+export function aggregateRecords(records: AirtableRecord[]) {
```

Comment:
@ewanmay can use this function to aggregate all records, just pass in all the records you want to aggregate (will probably be state.filtered_records in your case)